.github/workflows/pull-request.yml

```diff
@@ -12,7 +12,7 @@ env:
   NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 jobs:
   setup:
-    if: !github.event.pull_request.draft
+    if: github.event.pull_request.draft == false
```

Comment:
@micahjones13 no idea why this is suddenly needed...but it wouldn't run and gave an error: 

> unresolved tag !github.event.pull_request.draft

Reply:
lol yeah idk either. It was working with the "" around it, but in vscode it said that was bad so I'm good with this